std/format/write.d

```diff
@@ -79,129 +85,219 @@ module std.format.write;
     assert(w.data == "null");
 }
 
-/// Integrals are formatted like $(REF printf, core, stdc, stdio).
+/**
+Integrals are formatted in (signed) every day notation with `%s` and
+`%d` and as an (unsigned) image of the underlying bit representation
+with `%b` (binary), `%u` (decimal), `%o` (octal), and `%x` (hexadecimal).
+ */
 @safe pure unittest
 {
     import std.array : appender;
     import std.format.spec : singleSpec;
 
-    auto w = appender!string();
-    auto spec = singleSpec("%d");
-    formatValue(w, 1337, spec);
+    auto w1 = appender!string();
+    auto spec1 = singleSpec("%d");
+    formatValue(w1, -1337, spec1);
+
+    assert(w1.data == "-1337");
 
-    assert(w.data == "1337");
+    auto w2 = appender!string();
+    auto spec2 = singleSpec("%x");
+    formatValue(w2, -1337, spec2);
+
+    assert(w2.data == "fffffac7");
 }
 
-/// Floating-point values are formatted like $(REF printf, core, stdc, stdio)
+/**
+Floating-point values are formatted in natural notation with `%f`, in
+scientific notation with `%e`, in short notation with `%g`, and in
+hexadecimal scientific notation with `%a`. They are rounded according
+to the current rounding mode of the floating point unit, if available.
```

Comment:
```suggestion
to the current rounding mode of the floating point unit.
```

Reply:
If you want to keep the `if available` part you also need to specify what happens in the contrary case. If you go down that road, I suggest rewording so that the sentence starts with that: "If a rounding mode is available/provided [...], otherwise [...]."